log4j-core/src/main/java/org/apache/logging/log4j/core/util/datetime/FixedDateFormat.java

```diff
@@ -503,6 +510,15 @@ public String getFormat() {
         return fixedFormat.getPattern();
     }
 
+    /**
+     * Returns the length of the resulting formatted date and time strings.
+     *
+     * @return the length of the resulting formatted date and time strings
```

Comment:
This description is not correct, otherwise we won't need to double the size of the buffer in `InstantFormatter`.

Reply:
Right, I should add "in the ROOT locale" (which is basically English, but less anglocentric).

Reply:
I will appreciate that. Maybe also a note on _"if you want to determine the max. length, you better multiply this number by 2"_.